node/tests/integration/main.rs

```diff
@@ -418,7 +425,483 @@ async fn test_play_check() {
     execute_tx_and_sync(&mut client, tx_request).await;
 
     println!("Executed and synced with node");
-    assert_slot_status_check(&client, target_account_id, game_slot_data.clone()).await;
+    assert_slot_status_check(&client, target_account_id, game_slot_data.clone(), 1 as u8).await;
+}
+
+#[tokio::test]
+async fn test_e2e() {

```

Comment:
Since this test is pretty huge i think it would be better to have a different directory for these types of test

for example: `tests/e2e/test_basic_e2e.rs`